packages/icons/docs/stories/rainbowIconItem.js

```diff
@@ -0,0 +1,135 @@
+        isClickTooltip.current = true;
+    };
+
+    const handleTooltipMouseUp = () => {
+        isClickTooltip.current = false;
+        triggerRef.current.focus();
+    };
+
+    const handleTooltipMouseEnter = () => {
+        isHoverTooltip.current = true;
+    };
+
+    const handleTooltipMouseLeave = () => {
+        isHoverTooltip.current = false;
+        if (!isFocused) {
+            closeOverlay();
+        }
+    };
+
+    const handleKeyPressed = (event) => {
+        if (event.keyCode === ESCAPE_KEY) {
+            event.preventDefault();
+            closeOverlay();
+        }
+    };
+
+    const Icon = rainbowIcon.icon;
+    const IconName = rainbowIcon.name.charAt(0).toUpperCase() + rainbowIcon.name.slice(1);
+    const importLine = `import { ${IconName} } from ‘@rainbowmodules/icons’`;
+    const IconTag = `<${IconName} color=“#096666” size={32} />`;
+
+    return (
+        <>
+            <StyledButton
+                id={id}
+                ref={triggerRef}
+                onMouseEnter={openOverlay}
+                onMouseLeave={handleButtonMouseLeave}
+                onFocus={() => setIsFocused(true)}
+                onBlur={handleBlur}
+                onKeyDown={handleKeyPressed}
+                type="button"
+                ariaLabelledby={helpTextId}
+            >
+                <Icon isFocused={isFocused} />
+                <div>{rainbowIcon.name}</div>
+            </StyledButton>
+            <RenderIf isTrue={!!rainbowIcon}>
+                <InternalOverlay
+                    isVisible={isOpen}
+                    render={() => {
+                        return (
+                            <StyledTooltip
+                                id={helpTextId}
+                                role="tooltip"
+                                onMouseDown={handleTooltipMouseDown}
+                                onMouseUp={handleTooltipMouseUp}
+                                onMouseEnter={handleTooltipMouseEnter}
+                                onMouseLeave={handleTooltipMouseLeave}
+                            >
+                                <StyledTitle>Using rainbow/icons</StyledTitle>
+                                <StyledSample>Example</StyledSample>
+                                <StyledText>
+                                    <div>{importLine}</div>
+                                    <div>{IconTag}</div>
+                                </StyledText>
+                            </StyledTooltip>
+                        );
+                    }}
+                    triggerElementRef={triggerRef}
+                />
+            </RenderIf>
+        </>
+    );
+};
+
+RainbowIconItem.propTypes = {
+    /** Icons list with all existing icons in Rainbow Icons Collection. */
+    rainbowIcon: PropTypes.object,
+    /** id number for the box. */
+    id: PropTypes.number,
+};
+
+RainbowIconItem.defaultProps = {
+    rainbowIcon: null,
+    id: null,
```

Comment:
I see when you use this component in `packages/icons/docs/stories/rainbowIconsGallery.js` what you pass is ```iconBoxId={index}``` instead of id, if these ids are not needed then remove them